.github/workflows/docker-publish.yml

```
@@ -1,16 +1,118 @@
+            type=semver,pattern={{version}}
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create manifest list and push
+        working-directory: /tmp/digests
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:33: Quote this to prevent word splitting

(shellcheck)